host-configs/CVX/cvx-cpu.cmake

```diff
@@ -0,0 +1,17 @@
+site_name(HOST_NAME)
+set(CONFIG_NAME "cvx-cpu" CACHE PATH "" FORCE)
+message( "CONFIG_NAME=${CONFIG_NAME}" )
+
+set(CMAKE_C_COMPILER "/util/gcc/gcc-9.3.0/bin/gcc" CACHE PATH "" FORCE)
+set(CMAKE_CXX_COMPILER "/util/gcc/gcc-9.3.0/bin/g++" CACHE PATH "" FORCE)
+set(CMAKE_Fortran_COMPILER "/vend/intel/parallel_studio_xe_2020_update2/compilers_and_libraries_2020.2.254/linux/bin/intel64/ifort" CACHE PATH "" FORCE)
```

Comment:
use a `GCC_ROOT` to define the compilers here.

Reply:
done